src/PrestaShopBundle/Form/Admin/Improve/International/Language/LanguageType.php

```diff
@@ -66,90 +69,123 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         // That's why only < et > are forbidden (HTML).
                         'pattern' => '/^[^<>]+$/',
-                        'message' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
+                        'message' => $this->trans('This field is invalid', 'Admin.Notifications.Error'),
                     ]),
                 ],
             ])
             ->add('full_date_format', TextType::class, [
+                'label' => $this->trans('Date format (full)', 'Admin.International.Feature'),
+                'help' => $this->trans('Full date format (e.g., Y-m-d H:i:s).', 'Admin.International.Help'),
                 'constraints' => [
                     new NotBlank([
-                        'message' => $this->trans('This field cannot be empty', [], 'Admin.Notifications.Error'),
+                        'message' => $this->trans('This field cannot be empty', 'Admin.Notifications.Error'),
                     ]),
                     new Regex([
                         'pattern' => '/^[^<>]+$/',
-                        'message' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
+                        'message' => $this->trans('This field is invalid', 'Admin.Notifications.Error'),
                     ]),
                 ],
             ])
             ->add('flag_image', FileType::class, [
+                'attr' => [
+                    'class' => 'type-file',
+                ],
+                'label' => $this->trans('Flag', 'Admin.International.Feature'),
+                'help' => $this->trans('Upload the country flag from your computer.', 'Admin.International.Help'),
                 'required' => !$options['is_for_editing'],
                 'constraints' => [
                     new Image([
-                        'mimeTypesMessage' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
+                        'mimeTypesMessage' => $this->trans('This field is invalid', 'Admin.Notifications.Error'),
                     ]),
                 ],
             ])
             ->add('no_picture_image', FileType::class, [
+                'label' => $this->trans('"No-picture" image', 'Admin.International.Feature'),
+                'help' => $this->trans('Image is displayed when "no picture is found".', 'Admin.International.Help'),
                 'required' => !$options['is_for_editing'],
                 'constraints' => [
                     new Image([
-                        'mimeTypesMessage' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
+                        'mimeTypesMessage' => $this->trans('This field is invalid', 'Admin.Notifications.Error'),
                     ]),
                 ],
             ])
             ->add('is_rtl', SwitchType::class, [
+                'label' => $this->trans('Is RTL language', 'Admin.International.Feature'),
```

Comment:
```suggestion
                'label' => $this->trans('RTL language', 'Admin.International.Feature'),
```
@LouiseBonnard Do you know why there is " Is" here? I don't get it 🤔 Maybe I am wrong, but I think it should not be here.